systers/vms
# Administrator Tests

# Description
Added tests for administrator app
  - Uncommented tests
  - Migrated tests from automated-testing repo.

Fixes #685 

# Type of Change:
**Delete irrelevant options.**

- Code
- Quality Assurance


**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)


# How Has This Been Tested?
To test my tests:
```
python manage.py test administrator -v 2
```
To confirm non-breaking change:
```
python manage.py test -v 2
```

# Checklist:
**Delete irrelevant options.**


**Code/Quality Assurance Only**
